x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/utils.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import mustache from 'mustache';
+import { uniq, startCase, flattenDeep, isArray, isString } from 'lodash/fp';
+
+import {
+  ActionTypeModel,
+  AlertAction,
+  IErrorObject,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../../../../../../plugins/triggers_actions_ui/public/types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { TypeRegistry } from '../../../../../../../../../plugins/triggers_actions_ui/public/application/type_registry';
```

Comment:
Same as the other `import` of these types -- adding a comment here linking to an Alerting issue requesting these types be exposed will be helpful to future developers.